app/src/main/res/values-ru/strings.xml

```diff
@@ -10,4 +10,12 @@
     <string name="subscription">Подписка</string>
     <string name="subscription_status">Статус</string>
     <string name="subscribe">Подписаться</string>
+    <string name="TextView_password">Пароль</string>
+    <string name="auth_TextView_title">Авторизация</string>
+    <string name="auth_Button_login">Войти</string>
+    <string name="auth_TextView_haveNotRegisteredYet">Ещё не зарегистрированы?</string>
+    <string name="auth_TextView_register">Зарегистрироваться</string>
+    <string name="auth_Required_Fields_Empty_Error">Обязательные поля не заполнены</string>
```

Comment:
it's muck better to keep default `snake_case` for resources naming (not only strings, but layouts values and etc.)
`auth_Required_Fields_Empty_Error` -> `auth_required_fields_empty_error` and so on.